__tests__/src/pages/Home.test.js

```diff
@@ -0,0 +1,34 @@
+// Test generated by RoostGPT for test ReactUnitTesting using AI Type Open AI and AI Model gpt-4-1106-preview
+
+// Import React testing utilities
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import Home from "../../../src/pages/Home"; // Adjust the path based on your project structure
+
+// Mock framer-motion to avoid animation-related issues during testing
+jest.mock("framer-motion", () => ({
+  motion: ({ children, ...rest }) => <div {...rest}>{children}</div>,
+}));
+
+describe("Home Component", () => {
+  test("renders Home component with animated name and description", () => {
+    render(<Home />);
+
+    // Check if the animated name element is present
+    const nameElement = screen.getByText("Dipesh Malvia");
```

Comment:
Unable to find an element with the text: Dipesh Malvia. This could be because the text is broken up by multiple elements